{{ cookiecutter.__dirname }}/.pre-commit-config.yaml

```diff
@@ -24,19 +29,19 @@ repos:
         name: pylint (library code)
         types: [python]
         args:
-          - --disable=consider-using-f-string
+          - --disable=consider-using-f-string,wrong-import-order
         exclude: "^(docs/|examples/|tests/|setup.py$)"
       - id: pylint
         name: pylint (example code)
         description: Run pylint rules on "examples/*.py" files
         types: [python]
         files: "^examples/"
         args:
-          - --disable=missing-docstring,invalid-name,consider-using-f-string,duplicate-code
+          - --disable=consider-using-f-string,duplicate-code,invalid-name,missing-docstring,wrong-import-order
       - id: pylint
         name: pylint (test code)
         description: Run pylint rules on "tests/*.py" files
         types: [python]
         files: "^tests/"
         args:
-          - --disable=missing-docstring,consider-using-f-string,duplicate-code
+          - --disable=consider-using-f-string,duplicate-code,invalid-name,missing-docstring,protected-access,wrong-import-order
```

Comment:
sort and add `invalid-name` (tests have long names), `protected-access` (tests mess with protected things) and `wrong-import-order` (isort)